data/ui/ErrorPerspective.ui

```diff
@@ -68,15 +68,7 @@
     <property name="can_focus">False</property>
     <property name="title" translatable="yes">Uh Oh! Something went wrong…</property>
     <property name="has_subtitle">False</property>
-    <child>
-      <object class="GtkButton">
-        <property name="label" translatable="yes">Quit</property>
-        <property name="visible">True</property>
-        <property name="can_focus">True</property>
-        <property name="receives_default">True</property>
-        <signal name="clicked" handler="_on_quit_button_clicked" swapped="no"/>
-      </object>
-    </child>
+    <property name="show_close_button">True</property>
```

Comment:
Also remove the `_on_quit_button_clicked` method from `piper/errorperspective.py`.